tests/eas/acceptance.py

```diff
@@ -320,17 +320,17 @@ def test_first_cpu(self, experiment, tasks):
     @experiment_test
     def test_big_cpus_fully_loaded(self, experiment, tasks):
         """Offload Migration and Idle Pull: Big cpus are fully loaded as long as there are tasks left to run in the system"""
-        num_big_cpus = len(self.target.bl.bigs)
```

Comment:
Yeah, sounds good to just have one patch that updates the acceptance tests to remove big.LITTLE assumptions.